src/storage/src/source/mod.rs

```diff
@@ -870,6 +871,7 @@ pub(crate) fn create_source<G, S: 'static>(
     config: SourceConfig<G>,
     source_connector: &ExternalSourceConnector,
     aws_external_id: AwsExternalId,
+    persist_client: mz_persist_client::Client,
```

Comment:
hmm should we rename `Client` to `PersistClient` so you don't feel the need to prefix it in the code?

Reply:
ahh, I actually just copy-pasted lol -- but now that you mention it, could be a good idea!

Reply:
whoops, did this a while ago but forgot to link it https://github.com/MaterializeInc/materialize/pull/12134